backend/src/main/java/se/chalmers/dat265/group1/WebApi.java

```diff
@@ -127,13 +132,14 @@ private static void setupPhysicalInterface() {
 
     private static void setupOrderInterface() {
         //Orders
-        get(ORDERS_URL, (request, response) -> ci.getAllOrders(), gson::toJson);
-        get(ORDERS_URL + ORDER_ID_URL, (request, response) -> ci.getOrder(request.params(ORDER_ID_PARAM)), gson::toJson);
-        get(ORDERS_URL + ORDER_ID_URL + PARTS_URL, (request, response) -> ci.getOrderedParts(request.params(ORDER_ID_PARAM)), gson::toJson);
-        post(ORDERS_URL, ((request, response) -> ci.createNewOrder(gson.fromJson(request.body(), Order.class))), gson::toJson);
-        put(ORDERS_URL + ORDER_ID_URL, ((request, response) -> ci.updateOrder(request.params(ORDER_ID_PARAM), gson.fromJson(request.body(), Order.class))), gson::toJson);
-        post(ORDERS_URL + ORDER_ID_URL + PARTS_URL, ((request, response) -> ci.createNewOrderedPart(request.params(ORDER_ID_PARAM), gson.fromJson(request.body(), OrderedPart.class))), gson::toJson);
-        put(ORDERS_URL + ORDER_ID_URL + PARTS_URL + ORDEREDPART_ID_URL, ((request, response) -> ci.updateOrderDetail(request.params(ORDER_ID_PARAM), request.params("orderedPartID"), gson.fromJson(request.body(), OrderedPart.class))), gson::toJson);
+        get(ORDERS_URL, (request, response) -> oi.getAllOrders(), gson::toJson);
+        get(ORDERS_URL + ORDER_ID_URL, (request, response) -> oi.getOrder(request.params(ORDER_ID_PARAM)), gson::toJson);
+        get(ORDERS_URL + ORDER_ID_URL + PARTS_URL, (request, response) -> oi.getOrderedParts(request.params(ORDER_ID_PARAM)), gson::toJson);
+        post(ORDERS_URL, ((request, response) -> oi.createNewOrder(gson.fromJson(request.body(), Order.class))), gson::toJson);
+        put(ORDERS_URL + ORDER_ID_URL, ((request, response) -> oi.updateOrder(request.params(ORDER_ID_PARAM), gson.fromJson(request.body(), Order.class))), gson::toJson);
+        //post(ORDERS_URL + ORDER_ID_URL + PARTS_URL, ((request, response) -> oi.createNewOrderedPart(request.params(ORDER_ID_PARAM), gson.fromJson(request.body(), OrderedPart.class))), gson::toJson);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)